runtime/onert/core/include/compiler/train/CodeMap.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_COMPILER_TRAIN_CODE_MAP_H__
+#define __ONERT_COMPILER_TRAIN_CODE_MAP_H__
+
+#include <unordered_map>
+#include "compiler/CodeMap.h"
+#include "exec/train/TrainableFnSequence.h"
+#include "ir/train/ITrainableOperation.h"
+
+namespace onert
+{
+namespace compiler
+{
+namespace train
+{
+
+struct CodeAndInfo
```

Comment:
@ragmani (out of curiosity) You used `Train...` prefix for most cases. But `CodeAndInfo` and `CodeMap` don't have `Train...` prefix. I know it is not mandatory since they are defined in `train` namespace. But I am just curious.

Reply:
I didn't append the prefix because I implemented it without thinking deeply about naming.
I will append the prefix here as well to make it easier to find when modifying later.